### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by ya7on**

### CODE REVIEW

File: src/routes/search.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import src.bot
+import src.models
```
Comment (tmcgroul) [inline src/routes/search.py:1]: ModuleNotFoundError: No module named 'src.models'

**Review by tmcgroul — CHANGES_REQUESTED** [review]

File: run.sh [hunk 2]
```diff
@@ -1,3 +1,5 @@
->&2 echo "Bot starting"
+>&2 echo "Migrations"
```
Comment (ya7on) [inline run.sh]: It's outdated. I'm using noSQL

File: src/models.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# pylint: disable=maybe-no-member
+
+import datetime
+
+import mongoengine as mongo
+
+mongo.connect('project1', host='0.0.0.0', port=27017)
```
Comment (ya7on) [inline src/models.py:5]: Add customization

**Review by ya7on — COMMENTED** [review]
reminder for me

File: src/models.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# pylint: disable=maybe-no-member
+
+import datetime
+
+import mongoengine as mongo
```
Comment (tmcgroul) [inline src/models.py:3]: requirements.txt must contain mongoengine

File: src/models.py [hunk 5]
```diff
@@ -0,0 +1,79 @@
+# pylint: disable=maybe-no-member
```
Comment (tmcgroul) [inline src/models.py]: Not need

File: src/models.py [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+import datetime
+
+import mongoengine as mongo
+
+mongo.connect('project1', host='0.0.0.0', port=27017)
+
+
+class User(mongo.Document):
+    user_id = mongo.IntField(unique=True)
```
Comment (tmcgroul) [inline src/models.py]: Maybe specify some [field arguments](http://docs.mongoengine.org/guide/defining-documents.html#field-arguments)?

File: src/models.py [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    @staticmethod
+    def get(user):
+        try:
+            return User.objects.get(user_id=user.id)
+        except mongo.DoesNotExist:
+            return None
+
+
+class Queue(mongo.Document):
+    _type = mongo.StringField()
```
Comment (tmcgroul) [inline src/models.py]: Property `_type` not override built-in `type` func. So underscore for what?

File: src/utils/logger.py [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+        messages = json.loads(the_file.read())
+        answer = messages.get(LANG).get(name)
+        if answer:
+            return answer.format(*args)
+        else:
+            return name.format(*args)
+
+
+def info(text):
+    prefix = '[ {} ]'.format(datetime.now().strftime('%x %X'))
```
Comment (tmcgroul) [inline src/utils/logger.py]: Look on [python string interpolation](https://www.python.org/dev/peps/pep-0498/). Interpolation looks cool.

**Review by tmcgroul — CHANGES_REQUESTED** [review]

**Review by tmcgroul — APPROVED** [review]

**Pull request merged**